docs/docs/external/getting-started/test-network/setup-packages.md

```diff
@@ -0,0 +1,270 @@
+
+### Fabric Client (fabric-cli)
+
+The CLI is used to interact with a Fabric network, configure it and run chaincode transactions to record data on the channel ledger or query data. It is also used to interact with remote networks through the relay in order to trigger an interoperation flow for data request and acceptance.
+
+The `fabric-cli` source code is located in the `samples/fabric/fabric-cli` folder.
+
+#### Prerequisites
+
+If you are using a Linux system, make sure that lib64 is installed.
+
+_Note_: The setup and running instructions below were tested with all Node.js versions from v11.14.0 to v14.17.3.
+
+#### Installation
+
+You can install `fabric-cli` as follows:
+- Navigate to the `samples/fabric/fabric-cli` folder.
+- Create `.npmrc` from template `.npmrc.template`, by replacing `<personal-access-token>` with yours created [above](#package-access-token)..
+- Run the following to install dependencies:
+  ```bash
+  make build
+  ```
+- Use the `fabric-cli` executable in the `bin` folder for [subsequent actions](./ledger-initialization.md).
+
+### Fabric Relay
+
+The relay is a module acting on behalf of a network, enabling interoperation flows with other networks by communicating with their relays.
+The code for this lies in the `core/relay` folder.
+
+#### Running Relay in Host
+
+##### Building
+
+_Prerequisite_: make sure Rust is already installed and that the `cargo` executable is in your system path (after installation of Rust, this should be available in `$HOME/.cargo/bin`); you can also ensure this by running `source "$HOME/.cargo/env"`.
+
+Build the generic (i.e., common to all DLTs) relay module as follows:
+- Navigate to the `core/relay` folder.
+- Run the following:
+  ```bash
+  make
+  ```
+  
+##### Deployment
+
+An instance or a relay can be run using a suitable configuration file. Samples are available in the `core/relay/config` folder.
+
+Run a relay for `network1` as follows:
+- Navigate to the `core/relay` folder.
+- Run the following:
+  ```bash
+  RELAY_CONFIG=config/Fabric_Relay.toml cargo run --bin server
+  ```
+
+Run a relay for `network2` as follows (_do this only if you wish to test interoperation between the two Fabric networks `network1` and `network2`_)
+- Navigate to the `core/relay` folder.
+- Run the following:
+  ```bash
+  RELAY_CONFIG=config/Fabric_Relay2.toml cargo run --bin server
+  ```
+
+For more information, see the [relay README](https://github.com/hyperledger-labs/weaver-dlt-interoperability/tree/master/core/relay).
+
+
+### Fabric Driver
+
+A driver is a DLT-specific plugin invoked by the relay while channelling external data queries to the local peer network and collecting a response with proofs. The Fabric driver is built as a Fabric client application on the `fabric-network` NPM package.
+The code for this lies in the `core/drivers/fabric-driver` folder.
+
+#### Running Fabric Driver
```

Comment:
Same as above (remove this heading, increase size of subsequent headings)

Reply:
Fixed